### ISSUE TITLE
Hover Documentation

### ISSUE THREAD

**Issue by tingerrr** [typst/typst#1860]
### Description

Currently hovering over an element function will show its values throughout the document, this is helpful for debugging, but it would be great if it also showed some base info on the function itself, like its signature and description.

This could also link to the documentation itself to see more extensive things like examples and.

### Use Case

Being able to quickly look up the signature without having to go to the docs, searching for the function and checking its parameters will increase productivity.

Often times I'll look at a function forgetting the name for a parameter and having to look it up my self, the auto-complete doesn't help much if you forgot the name.

**Comment by laurmaedje** [comment]
Makes sense, but I wonder why the autocomplete doesn't help if you forgot the name of a parameter? If you hit Ctrl+Space in the argument list, it should show the names of all parameters (that aren't in the list yet).

**Comment by tingerrr** [comment]
Well, I didn't know that you could do that. After trying it out, it does help, but not on custom functions.

**Comment by BlueskyFR** [comment]
@laurmaedje I noticed the autocompletion is super buggy, like any new function or variable I create will not be completed during the first few minutes, idk if it is just me or a known bug
